.github/workflows/pathogen-repo-build.yaml.in

```diff
@@ -367,43 +367,56 @@ jobs:
         env:
           AWS_BATCH_JOB_ID: ${{ needs.run-build.outputs.AWS_BATCH_JOB_ID }}
         run: |
-          # When a running process is to be cancelled (e.g. due to either user
-          # request or job timeout), GitHub Actions sends it SIGINT and waits
-          # 7.5s for it to exit.¹  We immediately detach on interrupt, letting
-          # the next wait-N job in our chain pick it back up (if this job timed
-          # out) or the cancellation job (if by user request).
-          #
-          # ¹ <https://docs.github.com/en/actions/managing-workflow-runs/canceling-a-workflow#steps-github-takes-to-cancel-a-workflow-run>
-          exec nextstrain build \
+          # Using timeout to detach from build before we hit the job.timeout-minutes
+          # to avoid cancelling the job in order to avoid a "Cancelled" status
+          # for the whole workflow.
+          # <https://github.com/nextstrain/.github/issues/129#issuecomment-2956350306>
+          exit_status=0
+          timeout 359m nextstrain build \
             --aws-batch \
             --attach "$AWS_BATCH_JOB_ID" \
             --detach-on-interrupt \
```

Comment:
`timeout` sends a SIGTERM by default, which will result in the `nextstrain build` process terminating instead of cleanly detaching. This is probably ~fine, but since we intend to detach, it'd be nice to actually detach. Since we're using `--detach-on-interrupt`, we can have `timeout` send SIGINT instead with its `--signal INT` option.